### PR TITLE
Feat: Add Tableau Project env var

### PR DESCRIPTION
**Pull request by mzappitello**
We're now going to be publishing to tableau from staging in addition to prod. The Tableau Project is currently hardcoded to aim at the prod location. Create and use an environmental variable that stores the appropriate project for each instance of the publisher application.

Asana Task: [🚀 Stand up staging Publisher](https://app.asana.com/0/1205827492903547/1206391938568280/f)

### CODE REVIEW

**Review by rymarczy — APPROVED** [review]
:cake: Just need to make sure we get the devops PR merged for PROD before deploying this.

File: .github/actions/run_task/action.yaml
Comment (mzappitello) [inline .github/actions/run_task/action.yaml:1]: i created an action with a bash script to run the task.

when launching from the command line, you need to define the subnet and security group to be used. the bash script pulls both of them based on the task name and envirionment.

File: .github/workflows/run-task.yaml
Comment (mzappitello) [inline .github/workflows/run-task.yaml:1]: change this workflow to use the action.

File: src/lamp_py/runtime_utils/env_validation.py [hunk 1]
```diff
@@ -45,9 +45,10 @@ def validate_environment(
         value = os.environ.get(key, None)
         if value is None:
             missing_required.append(key)
-        # do not log private variables
-        if key in private_variables:
-            value = "**********"
+        else:
+            # do not log private variables
+            if key in private_variables:
+                value = "**********"
```
Comment (mzappitello) [inline src/lamp_py/runtime_utils/env_validation.py:51]: we were only logging private vars before 👎 . log them all now.

File: src/lamp_py/tableau/hyper.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def __init__(
         hyper_file_name: str,
         remote_parquet_path: str,
         lamp_version: str,
-        project_name: str = "LAMP API",
+        project_name: str = os.getenv("TABLEAU_PROJECT", ""),
```
Comment (mzappitello) [inline src/lamp_py/tableau/hyper.py:45]: project name is an env var now.

File: src/lamp_py/tableau/pipeline.py [hunk 3]
```diff
@@ -30,6 +30,7 @@ def start_hyper_updates() -> None:
             "TABLEAU_USER",
             "TABLEAU_PASSWORD",
             "TABLEAU_SERVER",
+            "TABLEAU_PROJECT",
```
Comment (mzappitello) [inline src/lamp_py/tableau/pipeline.py:33]: make sure we have a project name.

File: .github/actions/run_task/action.yaml
Comment (mzappitello) [reply .github/actions/run_task/action.yaml:1]: i'm still getting permissions errors when i run this. something about the way our permissions are structured for describing the vpcs is still giving us unauthed errors.

File: src/lamp_py/runtime_utils/env_validation.py (hunk 1 above)
Comment (rymarczy) [reply src/lamp_py/runtime_utils/env_validation.py:51]: I don't think this needs to be in an `else` block

the `metadata[key] = value` line below just needs to be un-indented one level. 

This was introduced here: https://github.com/mbta/lamp/commit/36b959265ff977425013732019088fe46082d827

when the `add_metadata` calls were removed.

**Review by rymarczy — APPROVED** [review]
One, suggestion. Otherwise, LGTM.

**Review by rymarczy — APPROVED** [review]
:cake:

**Pull request merged**